src/daemon/emitter.cc

```diff
@@ -653,13 +640,46 @@ bool Emitter::Reload(const proto::Configuration& conf) {
       optional->Wait();
       return optional->GetValue();
     };
-    Worker worker = std::bind(&Emitter::DoRemoteExecute, this, _1, resolver);
+    const ui32 shard =
+        remote.has_distribution() ? remote.distribution() : 0u;
```

Comment:
That would also mean a breakage in config files. Is it OK to check both for several releases and remove then?